scripts/k8s/setup-io-prereq.sh

```diff
@@ -198,11 +210,11 @@ if [ -n "$NVME_TCP" ]; then
   fi
 fi
 
-if [ -n "$INSTALL_ZFS" ]; then
+if [ -n "$SETUP_ZFS" ]; then
```

Comment:
Above lines 193-211 should be within mayastor function? Just like these 2 ifs here to zfs and lvm

Reply:
Done